src/Appwrite/Utopia/Database/Validator/Queries/Users/ListUsers.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Appwrite\Utopia\Database\Validator\Queries\Users;
+
+use Appwrite\Utopia\Database\Validator\Queries\Collection;
+
+class ListUsers extends Collection
```

Comment:
Class name could be just users unless we need support for something else that should be called `users*`
```suggestion
class Users extends Collection
```

Reply:
@stnguyen90, for general knowledge. When choosing vars, attributes, or classes names we usually avoid including the namespace in the name as we already get the context from the namespace.

Reply:
I wanted to use `List`, but it seems to be reserved. 

<img width="499" alt="image" src="https://user-images.githubusercontent.com/1477010/184971675-9445b6f5-5ee1-47d8-8d78-70c892315c1a.png">

Any suggestions?

Reply:
We can drop the `Users` directory and call the validator class `Users`, should be perfectly aligned with our conventions.

Reply:
How about later when we need multiple Validators per service?

Reply:
Good point. We can always adapt and change, it's better to not pre-optimize to every situation that might happen or might not. Anyways, most chances that this filtering will be per resource and not per service.

For example in storage we filter by files and buckets. Resources might appear in multiple services so they shouldn't necessarily be coupled under a service name. For example we might enable an endpoint to list the user's teams in the account service, and use the exact same filtering as we use for teams in the teams service. Another example might be our vars list that is now being listed under functions, but will also be listed under the project service for supporting global vars.

Reply:
> We can always adapt and change, it's better to not pre-optimize to every situation that might happen or might not.

Wouldn't we tackle the other resources/endpoints right after this endpoint? I'm hoping to clarify this stuff now so that whoever works on the next endpoints as a clear understanding of where/how to build the next Validator for the next endpoint.

Reply:
But I think looking at it as response models makes the most sense. So in src/Appwrite/Utopia/Database/Validator/Queries, well have a class for each response model (Files, Users, Collections, Databases, etc.). That sound good?

Reply:
Yes 👍

Reply:
The above has been applied.